bodhi/server/util.py

```diff
@@ -121,13 +121,14 @@ def generate_changelog(build: 'models.Build') -> typing.Optional[str]:
         return None
 
     # Grab the RPM header of the previous update, and generate a ChangeLog
-    oldtime = 0
```

Comment:
Hmm, I think this should stay, no? If you don't set it here, when `lastpkg` is `None`, `oldtime` will be unset, no?

Reply:
Bleh, just go with what you wrote above.

Reply:
(Yes you're right, but just go with what you wrote, it's cleaner)